packages/editor-ui/src/modules/communityNodes.ts

```diff
@@ -63,6 +91,40 @@ const module: Module<ICommunityNodesState, IRootState> = {
 				context.commit('setLoading', false);
 			}
 		},
+		async uninstallPackage(context: ActionContext<ICommunityNodesState, IRootState>) {
+			context.commit('setLoading', true);
+			try {
+				await uninstallPackage(context.rootGetters.getRestApiContext, context.getters.getCurrentModalPackage.packageName);
```

Comment:
`context.getters.getCurrentModalPackage.packageName` this should be a method param... that way it's reusable.